server/svix-server/src/main.rs

```diff
@@ -102,12 +102,12 @@ async fn main() {
     let mut app = Router::new()
         .nest("/api/v1", v1::router())
         .layer(TraceLayer::new_for_http().on_request(()))
-        .layer(AddExtensionLayer::new(pool.clone()))
-        .layer(AddExtensionLayer::new(queue_tx.clone()))
-        .layer(AddExtensionLayer::new(cfg.clone()));
+        .layer(Extension(pool.clone()))
+        .layer(Extension(queue_tx.clone()))
+        .layer(Extension(cfg.clone()));
```

Comment:
Are you sure this is correct? (really deprecated)
If so, can you please:
1. Split it to its own PR.
2. Reference in the commit where you got this information from (e.g. a link to the docs).

Reply:
I did something with Axum now (testing a fix for a bug I reported) and this deprecation seems to only be done in the to-be-released version, and not in the currently released one. No?

Reply:
No problem, I will remove from PR for now. `AddExtensionLayer` is deprecated since 0.4.7, which was released a few days ago: https://github.com/tokio-rs/axum/releases/tag/axum-v0.4.7

Reply:
Yeah, you're right, I dunno why it's not in the axum changelog. :|

https://github.com/tokio-rs/axum/blob/main/axum/CHANGELOG.md

Reply:
Please open a PR for it, happy to insta-merge it, we don't want the warnings. :)

Reply:
In any event, I should have updated Cargo.toml as well to explicitly reference the latest version :|

Reply:
Yeah, probably. :P 

Cargo "best practices" is not to commit the lock file for binaries, only libs (dunno whether we'll continue following this, but this is the recommendation I've seen). So I didn't get the update that showed it for you as deprecated. :) 

But yeah, just bump to 0.4.8 (I think that's the latest) and open a PR with this small change. Looking forward to having it in!

Reply:
I can't unsee it now because of you. Let me know if you prefer me to open a PR and I will. I want it fixed, haha. :)

Reply:
Raised #283

Reply:
Thanks!